### PR TITLE
Refine ProductView:

### PR DESCRIPTION
**Pull request by lesser1**
[Related Gropius Issue](https://frontend.gropius.duckdns.org/components/9d12d6ac-e85a-407c-bdff-a8321fb75d3a/issues/3265b3af-6a79-4706-9f3b-f746a7d56370)

Add readonly information regarding visibility.
Also change background color of extra menu only
for employees.

### Definition of Done
- [x] Requirements of the issue are met
- [x] Code has been reviewed
- [x] Code is documented

### CODE REVIEW

File: src/views/ProductView.vue [hunk 1]
```diff
@@ -1,21 +1,87 @@
...
                 <v-btn icon="mdi-arrow-left" @click="router.back()"></v-btn>
-                <v-spacer></v-spacer>
             </v-toolbar>
+            <v-card class="bg-grey-lighten-3" rounded="0" variant="flat">
+                <v-card-item>
+                    <v-card-title class="mr-1">
+                        Product: {{ product?.product.internalName }}
+                    </v-card-title>
+
+                    <v-card-subtitle
```
Comment (nk-coding) [inline src/views/ProductView.vue:13]: This formatting looks a little sus

Comment (nk-coding) [reply src/views/ProductView.vue:13]: Maybe a manual break after the > makes it look better without breaking anything frontend related?

File: src/views/ProductView.vue [hunk 2]
```diff
@@ -1,21 +1,87 @@
...
+                        Number of variants:
+                        {{ product?.product.variants.totalCount }}
+                    </p>
+                </v-card-text>
+            </v-card>
             <v-card class="bg-grey-lighten-4" rounded="0" variant="flat">
                 <v-card-item>
                     <v-card-title
-                        >Product Variant Version
+                        >Variant:
```
Comment (nk-coding) [inline src/views/ProductView.vue:39]: Same with these examples here.
However is this is the only working version feel free to ignore these comments

**Review by nk-coding — APPROVED** [review]
Well I don't know if a readonly switch would have been my UI element of choice, but you are the UI designer

**Comment by lesser1** [comment]
> Well I don't know if a readonly switch would have been my UI element of choice, but you are the UI designer

I totally get that. I thought about this a lot and then I realized I should think about it too long and then I settled on something and well, here we are...

**Pull request merged**